src/bridge.jl

```diff
@@ -55,7 +69,7 @@ function MOI.get(b::AbstractBridgeInstance, loc::MOI.ListOfConstraintIndices)
     for bridge in values(b.bridges)
         for c in MOI.get(bridge, loc)
             i = findfirst(locr, c)
-            if !iszero(i)
+            if i !== nothing && !iszero(i)
```

Comment:
It will be easier to clean this up later if the code is separated into different `if VERSION ...` blocks.

Reply:
Done in https://github.com/JuliaOpt/MathOptInterfaceUtilities.jl/pull/73